data/mods/Magiclysm/npc/TALK_OLD_MAGUS.json

```diff
@@ -20,7 +20,7 @@
       { "text": "Let's trade items.", "topic": "TALK_OLD_MAGUS", "effect": "start_trade" },
       {
         "text": "Can I do anything for you?",
-        "topic": "TALK_MISSION_OFFER",
+        "topic": "TALK_MISSION_LIST",
```

Comment:
I'm not convinced the change is necessary for this NPC.  There are other conditions applied to this response which might avoid the problem that `TALK_MISSION_OFFER` otherwise has.

Reply:
I tested it and this one works, but I have no problem to revert that part.

Reply:
I'm sure it works after the change; I have a bit more time today so I'll try testing to see whether it works before the change.

Reply:
This PR is very short so I don't think it's necessary to avoid this particular change even if it's technically useless